### ISSUE TITLE
Using this repository to source funding goals and sources

### ISSUE THREAD

**Issue by RichardLitt** [opensourcedesign/funding#10]
### What

Right now, there is not a clear place to discuss funding models. This repo is nominally listed as the place to do that, but there's some discussion and works already happening to move all of the content here to opensourcedesign/organization (see https://github.com/opensourcedesign/organization/issues/42). I think it would be better to have all of the content here, and to streamline how we open issues using better signposting. 

### How

I would like to propose the following:
- [ ] Edit the README to include a note saying that you should open issues if you know of a) an organization or group or project that could use funding, or b) if you know of a place where people could apply to get earmarked funding (which is different from money that comes from normal jobs or client work, I believe). 
- [ ] Include an `ISSUE_TEMPLATE.md` for GitHub specifying one of the two suggestions above, so that users could easily fill out forms given what they see. This may not be totally necessary, but it might help. 
- [ ] Using these labels: `meta` for discussions like this, `source` for a source of funding, and `recipient` (or something better? `project`?) for a project that can receive money. 

### What about our current slush fund? 

Currently, OSD has $450~ in funds yearly ([source](https://opencollective.com/opensourcedesign)). I think that the majority of this is lying fallow - some of it needs to be used to maintain the website URLs, but the rest isn't allotted. What I propose is that every month we divide the annual funds into monthly units (so, 1/12th of the money - roughly $35 a month) - and split that money among the projects people want to support. This would help us get our name out there, help make better FLOSS design tools, and help people feel happier that their money is going somewhere badass. 

#### How do we choose which project? 

Any easy way to do a poll on GitHub about how to allocate money is to make a GitHub issue each month with a comment for each open Funding issue (easy to find), and then have people thumbs up each comment as they see fit. We can then divide the money equally, by dividing up the number of thumbs per project versus the total number. There are other ways we could look into too.

### Why bother?

Because the more the money is used, the more likely it is that people will realize we do good work, and the more likely others are going to be keen to donate. 

### Why can't this all be done in the organization repo?

I think that this topic is too large and is an umbrella for a whole bunch of possible conversations about funding. I don't want to limit those to what people think fits within _organization_. 

### What about the website? 

Well, currently, opensourcedesign.net/organization/#funding isn't the most helpful, and opensourcedesign.net/organization/funding/ doesn't work or I can't find a link to it. I think that having clear sources here will make it easier to make a better website in the end. 

### Shouldn't we discuss this?

We did! On the call https://github.com/opensourcedesign/events/issues/49#issuecomment-279762039. We can discuss it further here, and in the next call. \o/

**Comment by simonv3** [comment]
Thanks for getting this started.

Of interest to @bnvk, @jancborchardt, @jdittrich, @ei8fdb, @evalica, etc.

**Comment by davelab6** [comment]
http://Opensource.guide has a funding section

**Comment by RichardLitt** [comment]
@davelab6 Thanks for pointing that out! What did you find useful in that guide? How can it be applied here?

**Comment by bnvk** [comment]
> Right now, there is not a clear place to discuss funding models. 

We've had this repo for almost two years. There's been minimal discussion
and minimal funding action. 

However, we have too many repos (at present) and people feel overwhelmed and
confused *where* to talk about *what* and *how* thus I'm trying to reduce this f
actor of OSD.

If we want to have a few **topic specific** repos for **discussions only**
with criteria like you suggest, OK, maybe that will work better. I dunno?!?

**Comment by RichardLitt** [comment]
> We've had this repo for almost two years. There's been minimal discussion
and minimal funding action.

I wouldn't say this is the fault of the repo. I'd say it's poor sign posting. Perhaps I am wrong. There may not be all that many funding opportunities for OSD. 

> However, we have too many repos (at present) and people feel overwhelmed and
confused *where* to talk about *what* and *how* thus I'm trying to reduce this f
actor of OSD.

Can you point to where people have stated this? Could we build a directory to help people navigate the repos? Repos are not that different from folders, and it ought to be possible to list them in a way that makes it easy what to talk about, where. Reducing repos doesn't necessarily solve this issue, although I agree that it is one solution. 

I am not sure that we're not conflating poor flow with too many repos, just like a website with many pages but no page listing or navigation headers. 

> If we want to have a few **topic specific** repos for **discussions only**
with criteria like you suggest, OK, maybe that will work better. I dunno?!?

Code could go here too! What we just need is a clear way of showing what each page is. Right now, some repos are on the website, some are (AFAIK), and it's difficult to know which. (I may be wrong here)

**Comment by simonv3** [comment]
From personal feedback I've received from people in the OSD organization I think the primary issue is that they automatically get added to a new repo and get all the notifications for it unless they remove unwatch the repo.

**Comment by equinusocio** [comment]
@simonv3 because people have also to "Unsubscribe" if are already subscribed to a discussion.

**Comment by simonv3** [comment]
@equinusocio yeah exactly - if we create repositories all over the place people get overloaded with notifications. That's part of the problem with the multiple repositories thing :/
